.jenkins/Jenkinsfile.appraisal

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/groovy
```

Comment:
for my understanding.. where's the glue that ties the above Jenkinsfile to using this specific `Jenkinsfile.appraisal` for the `build`?  (Since `job: APPRAISAL_JOB_NAME` doesn't match this file, so am just curious) Thanks!

Reply:
`APPRAISAL_JOB_NAME` is the name of the job that is configured [inside jenkins](https://jenkins.invocadev.com/job/Aggregate%20Gem/job/Appraisal%20Test/).  The `build` step is triggering a Jenkins build to run, not specifically another pipeline file.

Reply:
:ack: thanks! And if someone were to rename the job on jenkins, hopefully there is good error handling / the error will be obvious when this outer pipeline fails that that constant needs to be updated to match?

Reply:
Yea, it'll throw an error saying that it couldn't find a job by that name.  But you also can't rename jobs, you can only delete and recreate, so it's not as delicate since we shouldn't be deleting jobs.